app/controllers/discover.js

```diff
@@ -49,6 +49,7 @@ export default ApplicationController.extend({
     numberOfResults: 0,
     took: 0,
     numberOfSources: 0,
+    types: '',
```

Comment:
Any reason that this defaults to a string? An empty object or null would be a bit more clear.
In general try to keep a single type per variable, especially when setting a default value.